File: swig/src/tools/test_python/runtests.py
#!/usr/bin/python
"""
Run all regression tests of Python wrapper to TaLib
""" 

import unittest
from ta_defs import ta_defs_test
from ta_common import ta_common_test
from ta_func import ta_func_test


if __name__ == '__main__':
    unittest.main()

File: swig/src/tools/test_perl/runtests.pl
#!/usr/bin/perl
#
# Run all tests for Swig Perl wrapper module Finance::TA

use Test::Harness;

# Windows workaround - space problem
$^X =~ s/Program Files/PROGRA~1/;

@tests = qw( ta_defs.t ta_common.t ta_func.t ta_abstract.t );
runtests(@tests);

